app/src/handlers/census/MetagameEventEventHandler.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ *  ### CENSUS RESPONSE ATTRIBUTES ####
+ "event_name":"MetagameEvent",
+ "experience_bonus":"",
+ "faction_nc":"",
+ "faction_tr":"",
+ "faction_vs":"",
+ "metagame_event_id":"",
+ "metagame_event_state":"",
+ "timestamp":"",
+ "world_id":"",
+ "zone_id":""
+ * ### END ###
+ **/
+
+import { inject, injectable } from 'inversify';
+import EventHandlerInterface from '../../interfaces/EventHandlerInterface';
+import { GenericEvent } from 'ps2census/dist/client/utils/PS2Events';
+import { getLogger } from '../../logger';
+import config from '../../config';
+import { jsonLogOutput } from '../../utils/json';
+import MetagameEventEvent from './events/MetagameEventEvent';
+import { TYPES } from '../../constants/types';
+import AlertHandlerInterface from '../../interfaces/AlertHandlerInterface';
+
+@injectable()
+export default class MetagameEventEventHandler implements EventHandlerInterface {
+    private static readonly logger = getLogger('MetagameEventEventHandler');
+
+    constructor(
+        @inject(TYPES.AlertHandlerInterface) private alertHandler: AlertHandlerInterface
+    ) {
+    }
+
+
```

Comment:
Extranious line

Reply:
Then we should add the following lint rule:
"no-multiple-empty-lines": ["error",{ "max": 1, "maxEOF": 0 } ]

Reply:
Will likely go in as a result of #30, resolving for now as it'll eventually be dealt with.